### PR TITLE
add ability to create multiple queues in the same module

### PR DESCRIPTION
**Pull request by simonrondelez**
this changes the functionality of the module from expecting a string with 1 name for a sqs queue to a list of sqs names.

### CODE REVIEW

**Review by MattiasGees — COMMENTED** [review]
Do you want to have a seperate IAM policy per queue or are two (consumer, pusher) enough to cover all queues for your list?

**Comment by simonrondelez** [comment]
now it will create 2 aws_iam_policies per queue. I think thats best so its more isolated

**Review by Komorebi-E — APPROVED** [review]
All looks valid and makes sense.
A minor question about naming, the IAM policy names are in a different order to the queue mames . E.g. ENV is either first or last.

**Review by MattiasGees — CHANGES_REQUESTED** [review]
Can you also fix the outputs documentation as it will now be a list instead of a string.

File: sqs_with_iam/output.tf [hunk 1]
```diff
@@ -1,15 +1,15 @@
 output "arn" {
-  value = "${aws_sqs_queue.queue.arn}"
+  value = ["${aws_sqs_queue.queue.arn}"]
```
Comment (SamClinckspoor) [inline sqs_with_iam/output.tf]: afaik you'll need to output as `["${aws_sqs_queue.queue.*.arn}"]` for lists

File: sqs_with_iam/variables.tf [hunk 2]
```diff
@@ -2,7 +2,11 @@ variable "environment" {}
 
 variable "project" {}
 
-variable "name" {}
+variable "name" {
+  description = "list of the sqs queues"
+  type        = "list"
+  default     = []
```
Comment (SamClinckspoor) [inline sqs_with_iam/variables.tf]: its required, you don't want to set a default

**Review by SamClinckspoor — APPROVED** [review]

**Review by MattiasGees — APPROVED** [review]

**Pull request merged**